src/test-utils/mocks.ts

```diff
@@ -297,7 +299,14 @@ class MockPolymeshTransactionBase {
   blockNumber?: BigNumber;
   status: TransactionStatus = TransactionStatus.Unapproved;
   error: Error;
-
+  getTotalFees = jest.fn().mockReturnValue(
+    Promise.resolve({
+      total: new BigNumber(1),
+      payingAccountData: { account: { address: '0x06'.padEnd(66, '0') } },
```

Comment:
```suggestion
      payingAccountData: { account: { address: 'address' } },
```
Account address and identity DID are not same